src/main/java/org/redischool/attendance/details/CourseHelper.java

```diff
@@ -30,7 +30,7 @@ public String closestCourseDate(Date targetDate, CourseDetails courseDetails) {
     public List<Object> getFormattedDatesMap(CourseDetails courseDetails) {
         List<Object> datesMap = new ArrayList<>();
         for (int i = 0; i < courseDetails.getJavaDates().size(); i++) {
-            Map dateMap = new HashMap<String, List<String>>();
+            Map<String, Object> dateMap = new HashMap<>();
```

Comment:
why `Map<String, Object>` and not `Map<String, List<String>>` like it was on right side before?

Reply:
Fixed in 5d4cae9. It's actually not `Map<String, List<String>>` but `Map<String, String>`. You can see that in lines 41 and 42. In declaration in line 33 the type on the right hand side doesn't matter anyway because of type erasure, therefore it could have been wrong the whole time.